src/encoded/schemas/file.json

```diff
@@ -111,6 +111,7 @@
                     "Genome Region",
                     "Quality Control",
                     "Reference Genome",
+                    "RNA Expression",
```

Comment:
I am not sure if that is a correct label, maybe "RNA Quantification" is better, or "Expression Quantification"? or we can think something else, see if Elizabeth as some other idea for this label